### PR TITLE
Time machine README.md

### PR DESCRIPTION
**Pull request by AceHao**
Time machine README.md is added

### CODE REVIEW

**Comment by jenniferdai1** [comment]
add space between "machinegit"

File: README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+This is a time machinegit branch!
```
Comment (stefanv) [inline README.md]: Space missing between machine and git?

**Comment by Seth-Park** [comment]
nice

**Comment by AceHao** [comment]
Space is added in line 1, between machine and git.

**Comment by jenniferdai1** [comment]
:+1:

**Comment by 18epedersen** [comment]
:100:

**Comment by stefanv** [comment]
Looks good to me, merging!

**Pull request merged**